### ISSUE TITLE
about train question

### ISSUE THREAD

**Issue by alexliyang** [BoyuanJiang/Age-Gender-Estimate-TF#35]
just like total_loss = gender_loss + age_loss + L2_loss
use AdamOptimizer optimizer

but the gender accuracy is up to 0.90+, and age accuracy is only about 0.50+ 

how to solve this question?

**Comment by luboren** [comment]
I would suggest assign more weight to age_loss
